doc/admin/administration.md

```diff
@@ -9,7 +9,9 @@ control plane and will be documented in a future revision.
 
 Useful admin dmg commands to query NVMe SSD health:
 
-- Query Per-Server Metadata: `dmg storage query (list-devices|list-pools)`
+- Query Per-Server Metadata:
+  - `dmg storage query (list-devices|list-pools)`
+  - `dmg storage scan --nvme-meta` shows mapping of metadata to NVMe controllers
```

Comment:
I thought this might be a good place to add the note about 'During the period where stats are unavailable, dmg storage scan --nvme-meta will return ...' and 'dmg storage scan --nvme-health will fall back to displaying cached health stats..'

Reply:
that period of unavailability is eliminated in follow-up https://github.com/daos-stack/daos/pull/3528